src/metabase/driver.clj

```diff
@@ -615,6 +615,6 @@
 
 (defmulti db-start-of-week
   "Return start of week for given database"
-  {:added "0.37.0" :arlists '([driver database])}
+  {:added "0.37.0" :arlists '([driver])}
```

Comment:
this is completely unrelated, but while harmless, it was wrong

Reply:
shouldn't this be `:arglists` and not `:arlists` ?